backend/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "gradido-backend",
+  "version": "0.0.1",
+  "description": "Gradido unified backend providing an API-Service for Gradido Transactions",
+  "main": "index.js",
```

Comment:
src/index.js

Reply:
`src/index.ts`?
`build/index.js`?

Which one? `build/index.js` is not in the source repo but most likely would be included in a published package